simulation.py

```diff
@@ -0,0 +1,84 @@
+"""
+Fuctions for a process to simulate an alpha.
+"""
+
+import copy
+import importlib
+import sys
+from thousandaire.constants import DATA_LIST_ALL
+from thousandaire.constants import TRADING_CONFIG, TRADING_REGION
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+PRICE_DATASET = 'price_dataset'
+
+def initialize():
+    """
+    Initialize data we get.
+    """
+    raw_data = DataLoader(DATA_LIST_ALL).get_all()
+    workdays = {
+        region : raw_data['workdays'][region]
+        for region in TRADING_REGION if region in raw_data['workdays'].keys()}
+    return_data = {}
+    for key, value in workdays.items():
+        return_data[key] = copy.deepcopy(raw_data)
+        for dataset in return_data[key].values():
+            dataset.set_workdays(value)
+    return_data['workdays'] = workdays
+    return return_data
+
+def pure_data(raw_data, data_list, target, region):
+    """
+    Pure the data which alpha need.
+
+    Due to the raw_data is combined with all data we have, we need to pure
```

Comment:
Due to + N.
The object of this sentence is a sub-sentence, so please use "because" or "since", etc.